packages/tracing-web/src/index.ts

```diff
@@ -1,10 +1,10 @@
 import { agent, VERSION } from '@grafana/agent-core';
 import type { Instrumentation } from '@grafana/agent-core';
 
-export const tracingInstrumentation: Instrumentation = {
-  initialize: () => {
+export class TracingInstrumentation implements Instrumentation {
+  initialize(): void {
     agent.api.pushLog(['A simple log from the tracing instrumentation package']);
-  },
-  name: '@grafana/agent-tracing-web-instrumentation',
-  version: VERSION,
-};
+  }
+  name = '@grafana/agent-tracing-web:instrumentation';
```

Comment:
I know this may be annoying but order of members in a class should be: "variables" and then methods.

Reply:
- [ ] fix ordering